### ISSUE TITLE
Good Job!

### ISSUE THREAD

**Issue by AppNasty** [OneRice07/GoHANmem#13]
Not an issue but just wanted to say good job on GoHANmem so far. CANT wait till you add FUZZY search abilities. Its the LAST function needed to make any type of code on the fly. You need any help please dont hesitate to ask me. Ive been in game hacking scene for many years and will pitch in however i can. 

Any news on when you will release another update? What functions youre working on now? ANY NEWS at all will keep the hype up lol. 

-Sketch
OneHitGamerStudios.com

**Comment by OneRice07** [comment]
Thank u for using GoHANmem.
I'm testing FUZZY search,and it will be ok at the next release.

**Comment by AppNasty** [comment]
Awesome. Hey, if you need a tester im your guy. Using american Vita. Ima game developer and have done bug testing since i was 18 [34 now] for companies like Empire Interactive [creators of FlatOut racing series] and am a pro game hacker. I helped Aeolusc design CheatMaster for the PSP which turned into NitePR and CheatMaster Fusion. So id be glad to test functionality for you. And keep it secret until YOU release. Let me know.  Also, have you thought of adding ability to search for Letters and not just numbers? ie: 3F80? that would let us make size mods in games, speed, moon jumps etc as well.

**Comment by yushinyuma** [comment]
Nice., sketch.,i m already mem in ur forum., hope next for update have ability for fuzzy search and speed searching., like tempar 1.63 or anyone who can revise the tempar to vita., but your gohanmem will good to use., cant wait to update., tnx yeoo

**Comment by AppNasty** [comment]
Yeah man. Once Fuzzy hits in a matter of days i will have the most extreme
vita game hacks out for sure. You name it, it will be done.

**Comment by yushinyuma** [comment]
Bro can you post a tut to unlocked item using gohanmem?., i only know how to find inf,hp,mp,money,quantity of item that unlocked using fuzzy seach.,but now only money in vita age eheh

**Comment by AppNasty** [comment]
Changed site so much. Forums will be back eventually. I mainly operate on twitter now. @realOHG
